Sources/TuistKit/MCP/MCPServerCommandResolver.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import Mockable
+
+@Mockable
+protocol MCPServerCommandResolving {
+    func resolve() -> (String, [String])
+}
+
+struct MCPServerCommandResolver: MCPServerCommandResolving {
+    private let executablePath: String
+
+    init(executablePath: String = CommandLine.arguments[0]) {
+        self.executablePath = executablePath
+    }
+
+    func resolve() -> (String, [String]) {
+        if executablePath.contains("mise") {
+            return ("mise", ["x", "tuist@latest", "--", "tuist", "mcp"])
```

Comment:
why can't we also use the executable path directly in the case of mise as we're doing from the `Push build insights` post action?

Reply:
The post action is scoped to the project, where I think it makes sense to use the version of Mise that was used to generate the project. This is a global configuration, which is a bit more distant from the version of Tuist used with a project (through Mise), so I lean on using the latest version of the MCP server there.